--- font-bh-ttf-1.0.4/Makefile.am.orig	2023-02-28 00:44:32.000000000 +0100
+++ font-bh-ttf-1.0.4/Makefile.am	2023-03-01 22:24:35.120797756 +0100
@@ -37,7 +37,7 @@ fontdir = @FONTDIR@
 font_DATA = $(FONT_FILES)
 
 actualconfigdir = $(FC_CONFDIR)/conf.d
-availconfigdir = $(FC_CONFDIR)/conf.avail
+availconfigdir = $(datadir)/fontconfig/conf.avail
 dist_availconfig_DATA = 42-luxi-mono.conf
 
 EXTRA_DIST = $(FONT_FILES) COPYRIGHT.BH README.md
@@ -51,7 +51,7 @@ install-data-hook:
 	$(MKFONTDIR) $(DESTDIR)$(fontdir)
 	mkdir -p $(DESTDIR)$(actualconfigdir)
 	@rm -f $(DESTDIR)$(actualconfigdir)/$(dist_availconfig_DATA)
-	ln -s ../conf.avail/$(dist_availconfig_DATA) $(DESTDIR)$(actualconfigdir)
+	ln -s $(availconfigdir)/$(dist_availconfig_DATA) $(DESTDIR)$(actualconfigdir)
 	@RUN_FCCACHE@
 
 distuninstallcheck:
